ports/nuspell/portfile.cmake

```
@@ -1,11 +1,11 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO nuspell/nuspell
-    REF v3.1.1
-    SHA512 239855051d9f49ba16913283090c4214a8f6a6cc290d359ab54014ff76fc297c131b67c6748bd1d4cdcec43c00dccc7f0c1bf8b07e06c9c648bff52ff193e096
+    REF v4.0.1
+    SHA512 122616fd24c2de35000ba12d680d3617e7fe97881d33febbcb106ce70d3bac356a00d90680a1bc8ee631ed532ace0f4b4f9fce4760a25b0f7fc1d60553e74528
     HEAD_REF master
 
-    PATCHES cmake-disable-cli-and-docs.patch
+    PATCHES 0001-Disable-CLI-tool-and-docs.patch
```

```suggestion
    PATCHES cmake-disable-cli-and-docs.patch
```
Since the patch modifies the same problem as before. It would be better to use the original patch name.

---

Fixed.